react/features/salesforce/functions.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import { doGetJSON } from '../base/util';
+
+/**
+ * Fetches the recording link from the server.
+ *
+ * @param {string} url - The endpoint for the session records.
+ * @param {string} jwt - The JWT needed for authentication.
+ * @param {string} text - The search term for the session record to find.
+ * @returns {Promise<any>}
+ */
+export async function getSessionRecords(
+        url: string,
+        jwt: string,
+        text: ?string
+) {
+    const fullUrl = text ? `${url}/records?text=${text}` : `${url}/records/recents`;
```

Comment:
nit: given that they're two separate endpoints, which signify different operations (search / recents), I would say it would be cleaner to actually have two separate methods